src/fmu/sumo/sim2sumo/tables.py

```diff
@@ -87,9 +83,12 @@ def generate_table_meta(datafile, obj, tagname, config):
     """
     logger = logging.getLogger(__name__ + ".generate_table_meta")
 
-    metadata = generate_meta(
-        config, datafile, tagname, obj, SUBMOD_CONTENT.get(tagname, "property")
-    )
+    if "vfp" in tagname.lower():
+        content = "lift_curves"
+    else:
+        content = SUBMOD_CONTENT.get(tagname, "property")
```

Comment:
Is there any other way to determine if it is "vfp"?
My understanding is that, if possible, we want to avoid logic on tagname?

Reply:
Problem here is that vfp breaks the pattern. res2df gives us one table per submodule. Which then gets converted both to  tagname, and content. This is poor definition of metadata in general, but the only way we have found so far. But when it comes to vfp (which is really what is called lift curves, hence the content name, which is really something to do with how gas can help with getting more out of the wells, I dunno all the details), you can have many of these tables. Therefore we send in tagname, which is no longer exactly equal to submodule name, but e.g vfp_4. So the choice is doing it like it is solved here, or otherwise introduce another argument, which would exist solely for vfp, and defaulted for all the others, like so:

` def generate_table_meta(datafile, obj, tagname, config, content)`

Where content is directly transferable from tagname in most cases, but not for this one. Anyway. It is a mess, dunno..